### PR TITLE
Feature/get time according timezone

### PR DESCRIPTION
**Pull request by aussie-m-mike**
https://trello.com/c/CardPimT/349-add-eventtimeaccordingtimezoneattribute
https://trello.com/c/4c8ehuS9/350-add-colors-to-google-event
https://trello.com/c/T7YK49mg/351-add-attendee-reminders-to-google-event

### CODE REVIEW

**Review by dmitriy-uvin — APPROVED** [review]

File: api/app/Services/SocialAccount/Google.php [hunk 1]
```diff
@@ -93,15 +94,12 @@ public function revokeToken($token = null)
 
     public function createEvent(CalendarEventInterface $googleCalendarEvent): void
     {
-        if (!Auth::user()) {
-            throw new AuthenticationException();
-        }
-
-        $token = Auth::user()->googleAccounts[0]->token;
+        $token = User::findOrFail($googleCalendarEvent->getUserId())->googleAccounts[0]->token;
```
Comment (aLLeXUs) [inline api/app/Services/SocialAccount/Google.php:97]: Please use the repository for getting data from the database.

File: api/app/Entity/Event.php [hunk 2]
```diff
@@ -34,4 +35,11 @@ public function customFieldValues()
     {
         return $this->hasMany(CustomFieldValue::class, 'event_id');
     }
+
+    public function getEventTimeAccordingTimezoneAttribute()
```
Comment (aLLeXUs) [inline api/app/Entity/Event.php:39]: Return type missing.

File: api/app/Services/Calendar/Google/GoogleCalendarEvent.php [hunk 3]
```diff
@@ -9,34 +9,53 @@
...
         $this->startTime = $startTime;
         $this->duration = $duration;
+        $this->color = $color;
+        $this->atendeeEmail = $attendeeEmail;
         $this->summary = $summary;
         $this->location = $location;
         $this->description = $descritpion;
     }
 
+    private function getDefaultColor()
```
Comment (aLLeXUs) [inline api/app/Services/Calendar/Google/GoogleCalendarEvent.php:41]: Return type missing.

File: api/app/Services/Calendar/Google/GoogleCalendarEvent.php [hunk 4]
```diff
@@ -9,34 +9,53 @@
...
+    {
+        return GoogleCalendarColors::GOOGLE_EVENT_COLOR_BOLD_GREEN;
+    }
+
+    public function getUserId(): int
+    {
+        return $this->userId;
+    }
+
     public function getStartTime()
```
Comment (aLLeXUs) [inline api/app/Services/Calendar/Google/GoogleCalendarEvent.php:51]: Return type missing.

File: api/app/Services/Calendar/Google/GoogleCalendarEvent.php [hunk 5]
```diff
@@ -9,34 +9,53 @@
...
     {
-        return Carbon::parse($this->startTime)->toIso8601String();
+        return Carbon::parse($this->startTime)->toRfc3339String();
     }
 
     public function getEndTime()
```
Comment (aLLeXUs) [inline api/app/Services/Calendar/Google/GoogleCalendarEvent.php:56]: Return type missing.

File: api/app/Services/Calendar/Google/GoogleCalendarEvent.php [hunk 6]
```diff
@@ -53,4 +72,14 @@ public function getLocation(): ?string
     {
         return $this->location;
     }
+
+    public function getAttendeeEmail(): string
+    {
+        return $this->atendeeEmail;
+    }
+
+    public function getColorAccordingGoogle()
```
Comment (aLLeXUs) [inline api/app/Services/Calendar/Google/GoogleCalendarEvent.php:81]: Return type missing.

File: api/app/Services/Calendar/Google/GoogleCalendarEvent.php [hunk 7]
```diff
@@ -53,4 +72,14 @@ public function getLocation(): ?string
...
+    {
+        return GoogleCalendarColors::getColors()[$this->color] ?? $this->getDefaultColor();
```
Comment (aLLeXUs) [inline api/app/Services/Calendar/Google/GoogleCalendarEvent.php:83]: Please add a method for getting the color code by name.

**Review by aLLeXUs — CHANGES_REQUESTED** [review]

**Comment by aussie-m-mike** [comment]
Requested changes has been pushed

**Review by aLLeXUs — APPROVED** [review]

**Pull request merged**